### DISCUSSION TITLE
Integration with parcel?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [solidjs/solid#688] [Ideas]
With years of waiting, parcel v2 is finally here: https://parceljs.org/blog/v2/

It supports react and vue out of box. Will solid.js be part of them? (in some way).

**Comment by ryansolid** [comment]
We've been using Parcel 2 beta in codesandbox for a while. I imagine it just works. Mind you we still rely on Babel. Parcel doesn't really help us with SSR as they don't/can't transform JSX in node modules. I imagine it will continue to work for client side apps using Babel.

Did you have something else in mind?

**Reply by zhaoyao91** [reply]
> Mind you we still rely on Babel

What's the downside effect of this? lower compile speed?

**Reply by ryansolid** [reply]
Yes. It becomes the bottleneck for compile speed most likely.

**Reply by zhaoyao91** [reply]
IMO, parcel brings me two DX advantages:

- out of box
- speed

If solid has it's way to make great DX, parcel is not a must :)

**Reply by ryansolid** [reply]
Vite?

**Reply by zhaoyao91** [reply]
I've tried solid with vite, it's nice. But it also make use of babel instead of inside esbuild right?

**Reply by ryansolid** [reply]
Yeah not a shortcoming of Vite though, just that esbuild doesn't support custom AST transform and Solid's compiler is in babel. It's really the same thing everywhere.
